core/src/main/java/com/graphhopper/routing/util/EncodingManager.java

```diff
@@ -58,31 +59,25 @@
 
     /**
      * Instantiate manager with the given list of encoders. The manager knows several default
-     * encoders ignoring case.
-     *
-     * @param flagEncodersStr comma delimited list of encoders. The order does not matter.
+     * encoders using DefaultFlagEncoderFactory.
      */
-    public static EncodingManager create(String flagEncodersStr) {
-        return create(new DefaultFlagEncoderFactory(), flagEncodersStr);
+    public static EncodingManager create(String flagEncoders) {
+        return create(new DefaultFlagEncoderFactory(), Arrays.asList(flagEncoders.split(",")));
     }
 
-    public static EncodingManager create(FlagEncoderFactory factory, String flagEncodersStr) {
-        return createBuilder(parseEncoderString(factory, flagEncodersStr)).build();
+    public static EncodingManager create(FlagEncoderFactory factory, List<String> flagEncoderStrings) {
+        return createBuilder(flagEncoderStrings.stream().map(s -> parseEncoderString(factory, s)).collect(Collectors.toList())).build();
     }
```

Comment:
Previously we had a constructor function that took an encoder string and one that took an encoder string and a factory. But now we have one that takes an encoder string and one that takes a list of strings and a factory?

Reply:
I'm not happy with these methods to be honest. I'll revisit how to make it more consistent.

Reply:
Sure, that sounds useful especially if GraphHopper only wraps the functionality of EncodingManager.Builder anyway